### DISCUSSION TITLE
Positron starts but console doesn't get past the "Starting up..." message.

### DISCUSSION THREAD

**Discussion by slozan0** [posit-dev/positron#3707] [Q&A] [answered]
I just installed version 2024.06.1 build 27 on a Ubuntu 20.04.06. As the title mentions the app starts and shows what appears to be the correct state. But the console does no get pass the "Starting up.." message.

When creating a new project, in the "Set up project configuration" window, the app only displays "Discovering R versions...". 
I have a working version of Rstudio and I can start R from the terminal without issues. 

Any ideas?

**Comment by DavisVaughan** [comment]
Can you please copy and paste in your full `Positron R Extension` output logs? You can get there by hitting `Cmd + P` to pull up the command palette, typing in `Output: Show Output Channels...`, and clicking on `Positron R Extension`.

**Reply by slozan0** [reply]
I only see "Task" as a selection option.

Just FYI, in Linux to get to the command palette I used "Ctrl + Shift + P"

**Reply by juliasilge** [reply]
From the command palette, you'll want to choose "Show Output Channels" so you have these options:

![Screenshot 2024-06-30 at 3 05 59 PM](https://github.com/posit-dev/positron/assets/12505835/3950bb4a-7952-4077-ab10-0a85aa489e87)

And then choose "Positron R Extension" to see the logs. That will be helpful for us to understand why you can't get past "Starting up..."

**Reply by slozan0** [reply]
@juliasilge Unfortunately, there is no other option than "Tasks" (see image)
![Screenshot from 2024-07-02 10-13-24](https://github.com/posit-dev/positron/assets/1455437/ce6ceac1-3c53-4ef2-9ec8-24ae539adf41)

**Reply by juliasilge** [reply]
Whoa, I haven't seen that before! What about in this dropdown in the bottom panel:

![Screenshot 2024-07-02 at 10 22 09 AM](https://github.com/posit-dev/positron/assets/12505835/f7227dbb-11de-4673-9c79-60cb8e935744)

**Reply by slozan0** [reply]
I selected "OUTPUT" from the bottom panel, then clicked on the drop down menu. I only see "Tasks" (see image)
![Screenshot from 2024-07-02 11-11-38](https://github.com/posit-dev/positron/assets/1455437/ff77b883-0ed4-4dfc-932c-3fd8b040fee9)

**Reply by juliasilge** [reply]
Wow, something is definitely not going well here! Apologies that you are having a hard time.

Can you open Developer Tools (the command is "Toggle Developer Tools") and look for errors there, especially as related to the extension host? It will look something like this:

![devtools](https://github.com/posit-dev/positron/assets/12505835/568d0fd1-ae4a-4b46-b203-fd2f0778d148)

**Reply by slozan0** [reply]
All right! It looks like we are getting somewhere. Unfortunately it is a nasty one.

 ERR [uncaught exception in main]: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /usr/share/positron/resources/app/node_modules.asar.unpacked/@vscode/sqlite3/build/Release/vscode-sqlite3.node): Error: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /usr/share/positron/resources/app/node_modules.asar.unpacked/@vscode/sqlite3/build/Release/vscode-sqlite3.node)

**Reply by wesm** [reply]
My hunch is that Positron's Linux build environment (https://github.com/posit-dev/positron/blob/main/.github/workflows/build-release-linux.yml) is causing the glibc version requirement to be beyond some older Linux distributions, like Ubuntu 20.04. 

based on https://code.visualstudio.com/docs/supporting/FAQ, VSCode itself requires at least Ubuntu 20.04 / glibc >= 2.28

>[Can I run VS Code on older Linux distributions?](https://code.visualstudio.com/docs/supporting/FAQ#_can-i-run-vs-code-on-older-linux-distributions)
> Starting with VS Code release 1.86.1 (January 2024), VS Code desktop is only compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, or Ubuntu 20.04.
> If you are unable to upgrade your Linux distribution, the recommended alternative is to use our [web client](https://code.visualstudio.com/docs/editor/vscode-web). If you would like to use the desktop version, then you can download the VS Code release 1.85 from [here](https://code.visualstudio.com/updates/v1_85). Depending on your platform, make sure to disable updates to stay on that version. A good recommendation is to set up the installation with [Portable Mode](https://code.visualstudio.com/docs/editor/portable).

We can try fixing this by changing our release builds (including Ark https://github.com/posit-dev/ark/blob/main/.github/workflows/release-linux.yml) to use Ubuntu 20.04 cc @lionel-

**Reply by DavisVaughan** [reply]
See also https://github.com/microsoft/vscode/issues/203375#issuecomment-1927893504

**Reply by jmcphers** [reply, marked as answer]
I promoted this to an issue in Positron: https://github.com/posit-dev/positron/issues/3854
